cartographer_rviz/cartographer_rviz/drawable_submap.cc

```diff
@@ -54,9 +59,14 @@ DrawableSubmap::DrawableSubmap(const ::cartographer::mapping::SubmapId& id,
       scene_node_(display_context->getSceneManager()
```

Comment:
This still does not look like the latest version. There is no `scene_node_` anymore at master. It is named `submap_node_` instead.